server/routes/api.ts

```diff
@@ -191,13 +192,25 @@ router.post(
 	discountAPI('apply-discount', 'APPLY_DISCOUNT'),
 );
 
+// The two switch types are using different apis for now, membership to recurring contribution
+// is using the old api
 router.post(
-	'/product-move/:switchType/:subscriptionName',
+	'/product-move/to-recurring-contribution/:subscriptionName',
 	withOktaServerSideValidation,
 	productMoveAPI(
-		'product-move/:switchType/:subscriptionName',
+		'product-move/to-recurring-contribution/:subscriptionName',
 		'MOVE_PRODUCT',
-		['switchType', 'subscriptionName'],
+		['subscriptionName'],
+	),
+);
+// recurring contribution to supporter plus is using the new api
+router.post(
+	'/product-move/recurring-contribution-to-supporter-plus/:subscriptionName',
```

Comment:
Super minor - would it be worth pulling this route string out to a const so it can be referred to here and in the call to `productSwitchAPI`? They seem intrinsically linked looking at this and the other cases.

Reply:
Those all sound like fine reasons to leave it alone!